### PR TITLE
[CFE-486] Add a sidebar

### PR DESCRIPTION
**Pull request by danielbrzn**
This PR adds a responsive sidebar to the Style Guide for better usability.

Deployed to https://staging-bak-chor-mee.luxola.com

### CODE REVIEW

File: app/assets/stylesheets/sephora_style_guide/app/base.scss [hunk 1]
```diff
@@ -17,3 +39,142 @@ section {
...
+    transform: translateX(-100%);
+    margin-top: 60px;
+    width: 100%;
+    -webkit-overflow-scrolling: touch; // for momentum scrolling on iOS
+  }
+
+  &.show {
+    transform: translateX(0);
+  }
+
```
[the remark this answers is not in the document]
Comment (zamakkat) [reply app/assets/stylesheets/sephora_style_guide/app/base.scss]: Again, editorconfig should be fixing these.

File: app/views/sephora_style_guide/home/index.slim [hunk 2]
```diff
@@ -53,9 +77,25 @@ javascript:
       isOutOfStock: false,
       showAlertSuccess: false,
       showAlertWarning: false,
-      showAlertError: false
+      showAlertError: false,
+      showSidebar: false,
+      elements: [ { name: 'Colors', id: '#colors' }, { name: 'Typography', id: '#typography'}, { name: 'Buttons', id: '#buttons' }, { name: 'Input Fields', id: '#forms' }, { name: 'Tables', id: '#tables' }, { name: 'Grid', id: '#grid' } ],
```
Comment (zamakkat) [inline app/views/sephora_style_guide/home/index.slim]: Would be better indented. And you can consider declaring it as a var slightly up above for readability?

Comment (danielbrzn) [reply app/views/sephora_style_guide/home/index.slim]: Is what I've done alright? I indented it and declared it as a variable before the vue declaration.

Comment (zamakkat) [reply app/views/sephora_style_guide/home/index.slim]: Yeah better. Though having one on each line I think will be easier to follow:
```js
var elements = [
  { name: 'Colors', id: '#colors' },
  { name: 'Typography', id: '#typography'},
  // ...
]
```

**Pull request merged**